### PR TITLE
Add speakers page

### PR DESCRIPTION
**Pull request by PedroBailaAndrade**
- white space on hero image gone;
Captura de ecrã 2021-04-13, às 14 27 23

- naming of Speakers section inside schedule page section changed;
- redundant utility deleted;
- nav element updated so it doesn’t have additional div;

### CODE REVIEW

File: priv/site/schedule.slime [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+          span.Schedule-listTime 9:00-9:15
+          span.Schedule-listTalkTitle Introduction
+          span.Schedule-listSpeaker Alchemy Team
+
+  / .Marquee
+  /     = for _ <- 1..2 do
+  /       div
+  /         = for _ <- 1..2 do
+  /           span Talks
+  /           span 2021
```
Comment (joaojusto) [inline priv/site/schedule.slime:78]: Please delete commented code

File: priv/site/_includes/css/components/Schedule.scss [hunk 2]
```diff
@@ -0,0 +1,211 @@
+//temporary
+.MobileNav {
+  border-bottom: 1px solid var(--c-black);
+}
```
Comment (joaojusto) [inline priv/site/_includes/css/components/Schedule.scss:4]: Why is this here?

**Review by joaojusto — CHANGES_REQUESTED** [review]

File: priv/site/_includes/css/components/Schedule.scss [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+  color: var(--c-black);
+  font-size: var(--s-5);
+  line-height: var(--s-7);
+  text-transform: uppercase;
+  font-family: "Chakra Petch";
+  font-weight: 500;
+  border-top: 0;
+}
+
+.scheduleSpeakers-contentTalkTitle {
```
Comment (joaojusto) [inline priv/site/_includes/css/components/Schedule.scss:110]: I think this should be CamelCase, `ScheduleSpeakers-contentTalkTitle`.

File: priv/site/_includes/css/components/MobileNav.scss [hunk 4]
```diff
@@ -93,3 +92,14 @@
...
+.MobileNav-alternate {
+  color: var(--c-white);
+  border-bottom: 1px solid white;
+  
+  svg {
+    path {
+      fill: var(--c-white);
+    }
+  }
+}
```
Comment (joaojusto) [inline priv/site/_includes/css/components/MobileNav.scss:105]: Missing new line at end of file, you should take a look at your editor preferences, most likely you have the option to auto add the new line off.

File: priv/site/_includes/css/components/MobileNav.scss [hunk 5]
```diff
@@ -1,13 +1,12 @@
 .MobileNav {
-  --text-color: var(--c-white);
```
Comment (joaojusto) [inline priv/site/_includes/css/components/MobileNav.scss:2]: Can we get rid of the empty lines here?

File: priv/site/_includes/schedule/nav.slime [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+  div.entry
+    = link @env, "Home", to: "/"
+    = link @env, "Info", to: "/#info"
+  div.entry
+    span Alchemy Conf
+  div.entry
+    = link @env, "Schedule", to: "/schedule", style: "display: none"
+    = link @env, "Tickets", to: "/tickets"
+
+= include(@env, "_includes/mobile_nav.slime")
```
Comment (joaojusto) [inline priv/site/_includes/schedule/nav.slime:11]: Again, missing new line at end of file

File: priv/site/schedule.slime (hunk 1 above)
Comment (joaojusto) [reply priv/site/schedule.slime:78]: Please don't forget to remove the commented code

**Review by joaojusto — CHANGES_REQUESTED** [review]
This is good to go, we just need to fix the casing on some classes and the new lines at end of files.

File: priv/site/_includes/css/components/MobileNav.scss (hunk 5 above)
Comment (PedroBailaAndrade) [reply priv/site/_includes/css/components/MobileNav.scss:2]: I don't have that line.

**Review by joaojusto — APPROVED** [review]
LGTM 🚢 :shipit:

**Pull request merged**